core/common/src/test/java/tachyon/wire/BlockInfoTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.wire;
+
+import java.util.List;
+import java.util.Random;
+
+import org.codehaus.jackson.map.ObjectMapper;
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.google.common.collect.Lists;
+
+public class BlockInfoTest {
```

Comment:
A single test class, but without wire type classes. The reflection can find all classes under a package. I don't see why we need `checkEquality`, wouldn't using `.equals()` be enough? Yes, only createExample method is needed, but I think it's fine. In fact, if we ever want to generate docs for REST API in the future, then that method can be used for that too.

But as I said, I'm also fine with boilerplate for the time being, given we dont have too many structs anyway.